### ISSUE TITLE
Gossip Problem With Diplayed gossip menu options for 131 npc flag

### ISSUE THREAD

**Issue by Expecto** [TrinityCore/TrinityCore#16261]
1: We find the creature has a flag Questgiver 128 flag Gossip 1 flag QuestGiver 2 = total (131)
2 : Use Gossip
3: Gossip menu option not displayed (Example: Archmage Berinand) ID 25314

The same situation for all creatures with 131 Unit flag

Hash version:  e59b432  
DB - Last verision for 3.3.5a TDB

**Comment by Aokromes** [comment]
You know you can edit the tickets intead open close, open close, open? XD

**Comment by Expecto** [comment]
Problem with internet...

**Comment by Killyana** [comment]
And this issue is not valid as I updated all the npcs that have gossip menu + gossip option to force displaying the menu.
134217728   0x0800 0000	  CREATURE_TYPEFLAGS_FORCE_GOSSIP

**Comment by Expecto** [comment]
Please link commit where it was fixed.

**Comment by Killyana** [comment]
https://github.com/TrinityCore/TrinityCore/commit/90ef83ad052892ee3df56ccb9af9594b0dd8932f

**Comment by Saben65** [comment]
There is an issue with him, but like @Killyana said, that isn't it.

No idea what is going on with him. He wasn't talking to me so I started going through his npc_text, broadcast_text, gossip_menu, etc. The only thing I changed, was I changed the EmoteID0 of his broadcast_text (id 26192) from 0 to 1 and restarted. He started talking to me. To verify, I changed it back to 0 and restarted. I tried talking to him again, and he would talk to me still.
